v0/destinations/hs/transform.js

```diff
@@ -324,6 +340,17 @@ const processRouterDest = async inputs => {
   const errorRespList = [];
   // using the first destination config for transforming the batch
   const { destination } = inputs[0];
+  // reduce the no. of calls for properties endpoint
+  let propertyMap;
+  const traitsFound = inputs.some(input => {
+    if (getTraits(input.message)) {
+      return true;
+    }
+    return false;
+  });
```

Comment:
Let's simplify this logic a bit,

```suggestion
  const traitsFound = inputs.some(input => {
    getTraits(input.message) !== undefined
    }
  });
```

Reply:
simplified